src/Backend/Core/Installer/ModuleInstaller.php

```diff
@@ -1014,4 +1028,15 @@ private function getAndCopyRandomImage(): string
 
         return $randomName;
     }
+
+    private function getPageBlockTypeForModuleExtra(int $extraId): PageBlockType
+    {
+        $moduleExtra = BackendModel::getContainer()->get(ModuleExtraRepository::class)->find($extraId);
+
+        if (!$moduleExtra instanceof ModuleExtra) {
+            return PageBlockType::richText();
```

Comment:
Why don't you just return `null` here? Doing that make the code above a lot clearer imo.

Reply:
fixed in 5c00c2d